intranet/apps/eighth/views/admin/scheduling.py

```diff
@@ -147,7 +147,11 @@ def schedule_activity_view(request):
                             messages.error(request, f"Did not unschedule {name} because there is {count} student signed up.")
                         else:
                             messages.error(request, f"Did not unschedule {name} because there are {count} students signed up.")
-                    instance.save()
+
+                    if instance:
+                        instance.sticky_students.set(form.cleaned_data["sticky_students"])
+                        instance.members.add(*form.cleaned_data["sticky_students"])
```

Comment:
You also need to remove the user's current `EighthSignup` using `remove_signup` and create a new `EighthSignup` for the sticky activity. Just adding them to members causes bugs. It needs to be done the proper signup way (see the signup view for the proper way).

Reply:
In my latest commit [d726662](https://github.com/tjcsl/ion/pull/1723/commits/d726662fc6b13726af0b9c8e426365d226f613c5) I used `EighthSignup.objects.create_signup` in addition to adding the user to `self.members`. Is there anything else I should be aware of?